sentiment_analysis_tutorial.Rmd

```diff
@@ -0,0 +1,261 @@
+---
+title: "Sentiment Analysis Processing and Visualization Tutorial"
+author: "Lylybell Teran"
+output:
+  pdf_document:
+  html_document:
+    df_print: paged
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(warning = FALSE, message = FALSE, tidy.opts = list(width.cutoff = 60), tidy = TRUE)
```

Comment:
Also, make sure your format follows the following steps.
![1668461533479](https://user-images.githubusercontent.com/114460912/201769910-9dbf5c05-7f91-4541-bf4a-c70aec1254f3.png)